test/metabase/lib/aggregation_test.cljc

```diff
@@ -42,3 +48,143 @@
           (is-fn? op tag [venues-category-id-metadata] [venue-field-check]))
         (testing "with query/stage-number, return clause right away"
           (is-clause? op tag [venues-category-id-metadata] [venue-field-check]))))))
+
+(defn- aggregation-display-name [aggregation-clause]
+  (lib.metadata.calculation/display-name lib.tu/venues-query -1 aggregation-clause))
+
+(defn- aggregation-column-name [aggregation-clause]
+  (lib.metadata.calculation/column-name lib.tu/venues-query -1 aggregation-clause))
+
+(defn- field-clause
+  ([table field]
+   (field-clause table field nil))
+  ([table field options]
+   [:field (merge {:lib/uuid (str (random-uuid))} options) (meta/id table field)]))
+
+(deftest ^:parallel aggregation-names-test
```

Comment:
Most of these tests already existed, I just moved them so they live in the same namespace as the clauses they're testing